tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -461,7 +468,7 @@ private String constructExampleHtml(String xpath, String value, boolean nonTrivi
          */
         XPathParts parts = XPathParts.getFrozenInstance(xpath).cloneAsThawed();
         if (parts.contains("dateRangePattern")) { // {0} - {1}
-            result = handleDateRangePattern(value);
+            handleDateRangePattern(value, examples); // modified to add lines to examples, void
```

Comment:
seems like removing "result = " leads to test failures like:
`  TestExampleGenerator {
    Test4528
Error:  (TestExampleGenerator.java:957)  Error: : specifics: expected "〖❬1,23 ❭dollari delle Bermuda〗〖❬0,00 ❭dollari delle Bermuda〗", got null
`

Reply:
Yes, I've only modified one of the handlers so there are two incompatible implementations within the file so for the handlers I haven't modified tests will fail. I'll change the rest of the handlers and re-commit once I finish, and hopefully the tests will pass!

Reply:
I would delete the comment, "modified to add lines to examples, void" since that was about work in progress

Reply:
```suggestion
            handleDateRangePattern(value, examples);
```